pkg/crane/optimize.go

```diff
@@ -27,11 +27,12 @@ import (
 	"github.com/google/go-containerregistry/pkg/v1/remote"
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
 	"github.com/google/go-containerregistry/pkg/v1/types"
+	"k8s.io/apimachinery/pkg/util/sets"
```

Comment:
Not super happy about depending on this.

Reply:
It's fine :P